CMSgov/mint-app
# [EASI-4303] Status updates

# EASI-4303


## Changes and Description

- Extracted status changes into separate record
- Ensured model plan creating record is always first
- Removed readyForReview fields as they are covered by status changes
- Removed records for initial create of task list sections and op needs
- Updated the task list recent changes to reflect sorting
- Added unit tests



## How to test this change

Verify the changes listed above are reflected in the change history render
Tests pass

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
